server/repository/src/migrations/v2_08_00/add_donor_id_to_invoice_and_invoice_lines.rs

```diff
@@ -0,0 +1,27 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_donor_id_to_invoice_and_invoice_lines"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                ALTER TABLE invoice_line ADD COLUMN donor_default_id TEXT;
```

Comment:
I don't fully understand context of these columns - but it seems like invoice has a default donor ID, but then each line should have donor_id, specified ID for that line rather than the default for the whole invoice?

Also, wondering why `donor_default` and not `default_donor`? second (as per issue) reads a bit nicer to me?

Reply:
Yep confirmed with roxy, on invoice_line should just be donor_id 🙏

Reply:
> I don't fully understand context of these columns - but it seems like invoice has a default donor ID, but then each line should have donor_id, specified ID for that line rather than the default for the whole invoice?
> 
> Also, wondering why `donor_default` and not `default_donor`? second (as per issue) reads a bit nicer to me?

I agree, I prefer default_donor_id.  However I was reluctant to rename because OG uses 'donor_default_id' and I had found the renaming of fields to be confusing. 

What do you reckon?

Reply:
I've gone ahead with the rename to default_donor_id FYI :)

Reply:
Yeah I agree, rename it. Mindset being - OMS is the long term plan, we should do our best to make OMS make the most sense it can, even when it means a mapping from OG. Going forward, OMS will be standalone, and we'll no longer have to think about the renamings 🙏